docs/content/docs/2.components/select.md

```
@@ -668,6 +716,16 @@ name: 'select-items-avatar-example'
 ---
 ::
 
+### With badge in items
+
+You can use the `badge` property to display a [Badge](/docs/components/badge) inside the items.
+
+::component-example
+---
+collapse: true
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)